pkg/apis/externaldns/types.go

```diff
@@ -183,6 +185,7 @@ func (cfg *Config) ParseFlags(args []string) error {
 	// Flags related to Kubernetes
 	app.Flag("master", "The Kubernetes API server to connect to (default: auto-detect)").Default(defaultConfig.Master).StringVar(&cfg.Master)
 	app.Flag("kubeconfig", "Retrieve target cluster configuration from a Kubernetes configuration file (default: auto-detect)").Default(defaultConfig.KubeConfig).StringVar(&cfg.KubeConfig)
+	app.Flag("request-timeout", "Request timeout when calling Kubernetes APIs. 0 means no timeout").Default(defaultConfig.RequestTimeout.String()).DurationVar(&cfg.RequestTimeout)
```

Comment:
"Please add (default: 30s)" to the description.

Reply:
I think we can skip the `(default: 30s)`. It gets populated by the default value in the help:
 ```
...
--kubeconfig=""               Retrieve target cluster configuration from a Kubernetes configuration file (default: auto-detect)
 --request-timeout=30s         Request timeout when calling Kubernetes APIs. 0s means no timeout
...
```